code/ATMOSPHERICS/datum_pipeline.dm

```diff
@@ -115,10 +117,12 @@ var/pipenetwarnings = 10
 
 /obj/machinery/atmospherics/proc/addMember(obj/machinery/atmospherics/A)
 	var/datum/pipeline/P = returnPipenet(A)
-	P.addMember(A, src)
+	if(P) //Sanity checking
+		P.addMember(A, src)
 
 /obj/machinery/atmospherics/pipe/addMember(obj/machinery/atmospherics/A)
-	parent.addMember(A, src)
+	if(parent && A) //Sanity checking

```

Comment:
I don't think these should ever be `null`.

This is one of those cases of "papering over the problem". 

Sure, it prevents the runtime, but it doesn't address the underlying problem

Reply:
This is fair enough. I'll see if I can get to the root of the cause

Reply:
Okay I think this is fine if removed - I think it only runtimes if incorrectly mapped, anyway.